test/integration/targets/win_acl_inheritance/tasks/main.yml

```diff
@@ -0,0 +1,173 @@
+---
+ # Test setup
+- name: remove test folder for baseline
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}"
+    state: absent
+
+- name: create test folders
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}\\folder"
```

Comment:
Yeah, you're correct that it has to be quoted if it starts the value, but single quotes work too (and solve the escaping issue nicely). So many useful little dusty corners in YAML. :D